package.json

```diff
@@ -25,23 +25,23 @@
     "node": ">= 5.4.1 < 7"
   },
   "dependencies": {
-    "@angular/animations": "^4.0.0-rc.3",
-    "@angular/common": "^4.0.0-rc.3",
-    "@angular/compiler": "^4.0.0-rc.3",
-    "@angular/core": "^4.0.0-rc.3",
-    "@angular/forms": "^4.0.0-rc.3",
-    "@angular/http": "^4.0.0-rc.3",
-    "@angular/platform-browser": "^4.0.0-rc.3",
+    "@angular/animations": "^4.0.0-rc.5",
+    "@angular/common": "^4.0.0-rc.5",
+    "@angular/compiler": "^4.0.0-rc.5",
+    "@angular/core": "^4.0.0-rc.5",
+    "@angular/forms": "^4.0.0-rc.5",
+    "@angular/http": "^4.0.0-rc.5",
+    "@angular/platform-browser": "^4.0.0-rc.5",
     "core-js": "^2.4.1",
     "rxjs": "^5.0.1",
     "systemjs": "0.19.43",
     "zone.js": "^0.7.2"
```

Comment:
What about updating `zone.js` and `rxjs` as well? I can see some peerDependency issues.